lib/generators/suspenders/lint_generator.rb

```diff
@@ -0,0 +1,67 @@
+module Suspenders
+  module Generators
+    class LintGenerator < Rails::Generators::Base
+      include Suspenders::Generators::Helpers
+
+      source_root File.expand_path("../../templates/lint", __FILE__)
+      desc "Creates a holistic linting solution that covers JavaScript, CSS, Ruby and ERB."
+
+      def install_dependencies
+        run "yarn add stylelint eslint @thoughtbot/stylelint-config @thoughtbot/eslint-config --dev"
+      end
+
+      def install_gems
+        gem_group :development, :test do
+          gem "better_html", require: false
+          gem "erb_lint", require: false
+          gem "erblint-github", require: false
+          gem "standard"
+        end
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def configure_stylelint
+        copy_file "stylelintrc.json", ".stylelintrc.json"
+      end
+
+      def configure_eslint
+        copy_file "eslintrc.json", ".eslintrc.json"
+      end
+
+      def configure_prettier
+        copy_file "prettierrc", ".prettierrc"
+      end
+
+      def configure_erb_lint
+        copy_file "erb-lint.yml", ".erb-lint.yml"
+        copy_file "config_better_html.yml", "config/better_html.yml"
+        copy_file "config_initializers_better_html.rb", "config/initializers/better_html.rb"
+        copy_file "erblint.rake", "lib/tasks/erblint.rake"
+
+        if default_test_suite?
+          copy_file "better_html_test.rb", "test/views/better_html_test.rb"
+        elsif rspec_test_suite?
+          copy_file "better_html_spec.rb", "spec/views/better_html_spec.rb"
+        end
+      end
+
+      # TODO: Consider extracting this into Rails
+      def update_package_json
+        content = File.read package_json
+        json = JSON.parse content
+        json["scripts"] ||= {}
+
+        json["scripts"]["eslint"] = "npx eslint 'app/javascript/**/*.js'"
```

Comment:
This is awesome 🙌! I'm curious what your thoughts would be on aligning the script names and actions more closely with what is defined in the thoughtbot [ESLint-config blog post](https://thoughtbot.com/blog/eslint-configs-for-react-react-native#running-eslint)? That post matches how we often define these on JavaScript projects, including in [thoughtbelt](https://github.com/thoughtbot/thoughtbelt) (the React Native version of Suspenders). In particular:

* defining linter scripts with names `lint:{linter}` (eg. `lint:eslint`, `lint:stylelint`, `lint:prettier`)
* creating an additional `lint` script that runs _all_ linters, since we will most commonly want to run all linters.
* Adding a `lint:prettier` script, eg. `prettier --check '**/*' --ignore-unknown`
  * Do we want Prettier to run on JSON, CSS, and Markdown? It's my preference, but if not, we can restrict the glob a bit. The JSON formatting is definitely nice. I'm not sure offhand if we'll need CSS since we have stylelint. Markdown formatting is nice because it formats code in code blocks, but it can be a little annoying with tables, since it tries to line everything up.
* Running `eslint` with the `--max-warnings=0` flag. You can find more info about this in this [section of the blog post](https://thoughtbot.com/blog/eslint-configs-for-react-react-native#philosophy), but basically, we want warnings to also fail the build 
* No need for `npx` in the `eslint` and `stylelint` commands. Yarn or whatever package manager is used will find these since they're dev dependencies.
* I like to also have a `fix:prettier` command that runs Prettier and fixes any errors. CI would run `lint:prettier`. I rarely have to run `fix:prettier` in practice since my editor formats with Prettier on save, but it can be nice when we need to format files that were created by generators or for folks who don't have their editors configured with Prettier.